pkg/config/loader.go

```diff
@@ -2,10 +2,10 @@ package config
 
 import (
 	"github.com/Kong/kuma/pkg/core"
-	//"github.com/ghodss/yaml"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/pkg/errors"
-	"gopkg.in/yaml.v2" // todo(jakubdyszkiewicz) switch to "github.com/ghodss/yaml" when solved problems with loading xdsServer
+	// we use gopkg.in/yaml.v2 because it supports time.Duration
```

Comment:
I don't understand this.

Here is an example of `ghodds/yaml` doing marshalling/unmarshalling of `time.Duration` - https://play.golang.org/p/NdpmTyqitmw

Reply:
Sorry I wasn't clear enough. `ghodss` supports `time.Duration`, but you cannot deserialize `2s`, you have to write `2000000000` which I'd say is not user friendly at all

Reply:
There are workarounds for this, e.g. https://stackoverflow.com/questions/48050945/how-to-unmarshal-json-into-durations